pkgs/tools/system/zenith/default.nix

```
@@ -0,0 +1,23 @@
+{ stdenv, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "zenith";
+  version = "0.7.5";
+
+  src = fetchFromGitHub {
+    owner = "bvaisvil";
+    repo = pname;
+    rev = version;
+    sha256 = "1hybw4nqgv2l8l4vfcsk8db540a66sh7sb32byhgpcky154zasal";
+  };
+
+  cargoSha256 = "1wfk7v70c7xqfcnlp9fmv7cs8d6ydh90502s5p15zy2qrsvan73p";
+
+  meta = with stdenv.lib; {
+    description = "Sort of like top or htop but with zoom-able charts, network, and disk usage";
+    homepage = https://github.com/bvaisvil/zenith;
```

RFC 0045 deprecated unquoted URLs

```suggestion
    homepage = "https://github.com/bvaisvil/zenith";
```